orc8r/cloud/helm/orc8r/templates/controller.deployment.yaml

```diff
@@ -122,7 +122,7 @@ spec:
             - name: HELM_RELEASE_NAME
               value: {{ .Release.Name }}
             - name: SERVICE_REGISTRY_NAMESPACE
-              value: {{ .Release.Name }}
+              value: {{ .Release.Namespace }}
```

Comment:
Can you explain this change fmu

Reply:
It's an env var we use to know what namespace the service-registry service is in. I had set this incorrectly, but didn't realize it as we coincidently use `orc8r` as both the release name and namespace for our deployments. This caused issues when deployed with minikube.